README.md

```diff
@@ -9,7 +9,7 @@
 ## Description
 The module installs and configures the Open Monitoring Distribution (OMD) which includes Nagios, check_mk and lots of other tools. Beside the server component, the module is also able to install and configure the check_mk agent.
 
-Original coded by erwbgy, forked gnubila-france and finally taken over by Voxpupuli.
```

Comment:
I wasn't too comfortable with the wording `taken over by`.  To me, it inadvertently came across a bit aggressive, when in fact, the migration has the [blessing](https://github.com/gnubila-france/puppet-check_mk/pull/13#issuecomment-474273593) of the previous owner.